apitools/base/py/credentials_lib.py

```diff
@@ -36,8 +33,13 @@
 from apitools.base.py import exceptions
 from apitools.base.py import util
 
+# pylint: disable=wrong-import-order,ungrouped-imports
+try:
+    from oauth2client import gce, locked_file, multistore_file
```

Comment:
Add a comment about the versions this spans and at what point it's safe to remove?